ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py

```diff
@@ -0,0 +1,42 @@
+# global
+from hypothesis import given
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# asarray
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")
+    ),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
```

Comment:
We actually don't need to use another strategy to get `dtype`, we can just pass it from `dtype_and_x`

Reply:
Well